content/02.intro.md

```diff
@@ -1,11 +1,11 @@
 ## Introduction {#intro}
 
-Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
+Deep learning (DL), a subfield of machine learning (ML) focusing on artificial neural networks with many layers, which are increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
```

Comment:
I think switching from "is" to "which are" introduces a grammatical error. Removing the appositive, we have

 > Deep learning (DL) which are increasingly used for the analysis of biological data

Reply:
Hm, with "which are" I was referring to "artificial neural networks with many layers" so you suggest it should change it to refer to "deep learning" instead?

Reply:
If the "which are" is referring to "artificial neural networks with many layers", then the subject of the sentence ("Deep learning") doesn't have a verb. In that case, I think it should be:

> Deep learning (DL) is a subfield of machine learning (ML) focusing on artificial neural networks with many layers, which are increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].

or

> Deep learning (DL), a subfield of machine learning (ML) focusing on artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].

I am a fan of the former, but the latter works equally well from a logical standpoint since we say that DL == a subfield of machine learning (ML) focusing on artificial neural networks with many layers

Reply:
ohhh, you are right. Sorry about that. For some reason, in my head, I was reading it as

> Deep learning (DL) is a subfield of machine learning (ML) focusing on artificial neural networks with many layers, which are increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].

all along.

... maybe it's time that I finally need glasses (or spring break :P). Will update.